src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json

```diff
@@ -34,5 +34,12 @@
             "priority":9996
         },
         "OP":"SET"
+    },
+    {
+        "ACL_RULE_TABLE:dataacl:default_rule":{
+            "PACKET_ACTION":"DROP",
```

Comment:
The [implicit rule](http://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus3000/sw/security/503_u3_1/b_Cisco_n3k_Security_Config_503_u3_1/b_Cisco_n3k_Security_Config_503_u3_1_chapter_01011.html#con_1300782) is to drop all IP and IPv6 packets.